pages/news/2020-05-15-you-dont-need-to-pay-for-mixxx.html

```diff
@@ -0,0 +1,76 @@
+title: You don't need to pay for Mixxx!
+author: Holzhaus
+date: 2020-05-15 13:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+
+Over the past few years there have been multiple instances of sellers that distribute paid boxed versions of Mixxx on eBay and Amazon.
+While it's not illegal to do this per se, we are not affiliated with these sellers and neither earn money from this nor endorse this in any way.
+Instead, you can download the latest version of Mixxx from the [download page on our website]({% url "/download.html" %}) for free.
+
+If you want to support Mixxx financially, consider [donating](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=QSFMYWN2B3JD2&source=url) - we'll appreciate it!
+
+
+### Mixxx was, is and always will be free
+
+In contrast our commercial competitors that offer "free" featured-reduced versions of their Software, the Mixxx developers do not impose any artificial restrictions on their software.
+There is no "Pro Edition" or Premium version of Mixxx, even if it says so in the description of the online shop.
+
+The Mixxx developers don't sell Mixxx, and if you "buy" Mixxx somewhere online you'll likely just get the free Mixxx version that the seller downloaded from our website and burned onto a CD.
+Even worse, some of these paid Mixxx versions might even contain ads or malware according to Amazon reviews.
+Just use the free [download]({% url "/download.html" %}) from our website to get the ad-free, malware-free, most recent version of Mixxx.
+
+
+### Mixxx is free as in freedom, not just free as in "free beer"
+
+You don't need to pay money to use Mixxx, but there's more:
+Its source code is accessible to everyone and if you know how to code, you can modify it and customize Mixxx to a degree that is simply not possible with any closed-source software.
+Even if you did get your hands on the source code of Serato DJ Pro/Lite, Rekordbox, Traktor or VirtualDJ (which is not available their users), it would be illegal for you to modify the program or redistribute it.
+
+Mixxx, on the other hand, is [free software](https://www.gnu.org/philosophy/free-sw.html) and grants you the four essential freedoms:
+
+- The freedom to run the program as you wish, for any purpose.
+- The freedom to study how the program works, and change it so it does your computing as you wish. Access to the source code is a precondition for this.
+- The freedom to redistribute copies so you can help others.
+- The freedom to distribute copies of your modified versions to others. By doing this you can give the whole community a chance to benefit from your changes. Access to the source code is a precondition for this.
+
+These freedoms are part of the license that Mixxx is subject to - the [GNU General Public License (GPL)](https://github.com/mixxxdj/mixxx/blob/master/LICENSE).
```

Comment:
Thanks to the GPL, these freedoms can never be taken away from you. The GPL is a copyleft license, meaning anyone who distributes or modifies Mixxx has to use the same license and provide the source code. The original Mixxx developers chose the GPL when they started Mixxx back in the early 2000s. They stopped working on Mixxx many years ago, but thanks to the copyleft terms of the GPL, you have the same freedoms with Mixxx now as when the first version of Mixxx was published. We can't take that away, nor could any future developers, nor could any company. It's free because it's yours.